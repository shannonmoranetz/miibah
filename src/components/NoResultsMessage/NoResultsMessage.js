import React from 'react';

export const NoResultsMessage = () => {
  return (
    <div className="NoResultsMessage">
      <h2>sorry!</h2>
      <p>
        the search returned no results.
        please try again with a different query.
      </p>
      <button onClick={() => window.location.reload()}>back home</button>
    </div>
  );
}

export default NoResultsMessage;