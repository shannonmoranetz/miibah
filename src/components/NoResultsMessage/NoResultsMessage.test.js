import React from 'react';
import { NoResultsMessage } from './NoResultsMessage';
import { shallow } from 'enzyme';

describe('NoResultsMessage', () => {
  let wrapper; 
  beforeEach(() => {
    wrapper = shallow(<NoResultsMessage />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});