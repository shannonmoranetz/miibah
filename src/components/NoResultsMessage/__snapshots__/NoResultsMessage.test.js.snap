// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResultsMessage should match the snapshot 1`] = `
<div
  className="NoResultsMessage"
>
  <h2>
    sorry!
  </h2>
  <p>
    the search returned no results. please try again with a different query.
  </p>
  <button
    onClick={[Function]}
  >
    back home
  </button>
</div>
`;
